lib/routes/routeBackbeat.js

```diff
@@ -198,18 +265,31 @@ function routeBackbeat(clientIP, request, response, log) {
                 log.debug('no such route', { method: request.method,
                     bucketName: request.bucketName,
                     objectKey: request.objectKey,
-                    resourceType:
-                                             request.resourceType });
+                    resourceType: request.resourceType
+                });
                 return next(errors.MethodNotAllowed);
             }
-            const versioningConfig = bucketInfo.getVersioningConfiguration();
-            if (!versioningConfig || versioningConfig.Status !== 'Enabled') {
-                log.debug('bucket versioning is not enabled',
-                    { method: request.method,
+            if (usingMultipleBackend) {
```

Comment:
This can be combined into one
```js
 if (usingMultipleBackend && !_isValidLocation(request)) {

}
```

Reply:
I want to return the route if using a multiple backend, and an error if the location is invalid.

Reply:
I see the login one, missed it during the review.